raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -703,6 +702,37 @@ def events_for_unlock_if_closed(
     return events
 
 
+def events_for_expired_locks(
+        mediator_state: MediatorTransferState,
+        channelidentifiers_to_channels: typing.ChannelMap,
+        block_number: typing.BlockNumber,
+        pseudo_random_generator: random.Random,
+):
+    events = list()
+
+    transfer_pair: MediationPairState
+    for transfer_pair in mediator_state.transfers_pair:
+        balance_proof = transfer_pair.payee_transfer.balance_proof
+        channel_identifier = balance_proof.channel_identifier
+        channel_state = channelidentifiers_to_channels.get(channel_identifier)
+
+        assert channel_state
```

Comment:
Can you add a message to this assertion? Something like `'Couldn't find channel for channel_id: {}'.format(channel_identifier)`